staging/src/k8s.io/cli-runtime/pkg/genericclioptions/config_flags.go

```diff
@@ -303,7 +313,9 @@ func (f *ConfigFlags) AddFlags(flags *pflag.FlagSet) {
 	if f.Timeout != nil {
 		flags.StringVar(f.Timeout, flagTimeout, *f.Timeout, "The length of time to wait before giving up on a single server request. Non-zero values should contain a corresponding time unit (e.g. 1s, 2m, 3h). A value of zero means don't timeout requests.")
 	}
-
+	if f.ClientCacheTTL != nil {
+		flags.StringVar(f.ClientCacheTTL, flagClientCacheTTL, *f.ClientCacheTTL, "The TTL (in minutes) for client-go discovery cache")
```

Comment:
include the default in the help text

Reply:
Also, I don't think the duration is actually "in minutes".